src/Control Tasks/RockblockControlTask.cpp

```diff
@@ -122,7 +122,7 @@ void RockblockControlTask::dispatch_standby()
 void RockblockControlTask::dispatch_send_at()
 {
     sfr::rockblock::conseq_reads = 0;
-#ifdef VERBOSE
+#ifdef VERBOSE_IMUD
```

Comment:
Either add the regular VERBOSE flag additionally here or make sure everyone environment that has the VERBOSE flag also has the VERBOSE_IMUD flag